src/DataFixtures/GroupSeeder.php

```diff
@@ -36,7 +54,17 @@ public function load(ObjectManager $manager)
             };
             $group->setName($name);
             $group->setSlug(Slug::slugify($name));
-            $group->setIsVisible($faker->boolean(75));
+
+            // isVisible property. We might need to force the last groups to be visible or not, so that the minimum/maximum amounts are respected.
+            $isVisible = true;
+            if ($this->minimumVisibleGroupCount - $visibleGroupCount < 19 - $i // Minimum condition
+                && $this->maximumVisibleGroupCount !== $visibleGroupCount) {   // Maximum condition
+                $isVisible = $faker->boolean(75);
+            }
+            if ($isVisible) {
+                ++$visibleGroupCount;
+            }
+            $group->setIsVisible($isVisible);
```

Comment:
Ton algorithme est trop complexe pour la simple tâche à réaliser.
Je te propose de mettre les premiers groupes en visible pour atteindre le minimum, puis faire le générateur à 75% sans dépasser le max.
```suggestion
            // isVisible property. We might need to force the last groups to be visible or not, so that the minimum/maximum amounts are respected.
            if ($visibleGroupCount < $this->minimumVisibleGroupCount) {
                $isVisible = true;
            } else if ($visibleGroupCount < $this->maximumVisibleGroupCount) {
                $isVisible = $faker->boolean(75);
            } else {
                $isVisible = false;
            }
            
            if ($isVisible) {
                $visibleGroupCount++;
            }
            $group->setIsVisible($isVisible);
```